pkg/validate/validate_test.go

```diff
@@ -49,6 +49,47 @@ func TestClusterName(t *testing.T) {
 	}
 }
 
+func TestClusterName1035(t *testing.T) {
```

Comment:
Thanks for adding this test?

Reply:
Was using that function and it didn't have a test so thought I'd add it.